README.md

```diff
@@ -9,6 +9,10 @@ Clone the Brain Observatory Toolbox repository using `git`, or download a releas
 
 Some aspects of the toolbox make use of accelerated `.mex` files, and therefore require `mex` to be configured. These files will be compiled automatically when needed; if `mex` is not available, then `matlab` native versions will be used.
 
+#### Privacy information
+
+We track the usage of this toolbox along with some demographics, for us to understand the demand for the toolbox.
```

Comment:
@DylanMuir  Perhaps we should say:  'We track the usage of this toolbox along with some demographics, for us to understand the demand for the toolbox using the Google Analytics API.'  so that the users will be away we are tracking the usage of the toolbox over the web.